templates/deployment.yaml

```diff
@@ -116,6 +116,10 @@ spec:
               subPath: '{{ .filename }}'
               {{- end }}
             {{- end }}
+          {{- with .Values.deployment.container.resources }}
+          resources:
```

Comment:
This will be useful to add to the default [values.yaml](https://github.com/drinkataco/media-servarr/blob/main/values.yaml) file for documentation, similarly to how other things (such as livenessProbe) are documented: https://github.com/drinkataco/media-servarr/blob/main/values.yaml#L73-L79